wgsl/index.bs

```diff
@@ -2860,12 +2860,12 @@ Proposal: pipeline creation fails with an error.
 
 <div class='example wgsl global-scope' heading='Module constants, pipeline-overrideable'>
   <xmp>
-    [[override(0)]]    let has_point_light : bool = true;  // Algorithmic control
-    [[override(1200)]] let specular_param : f32 = 2.3;     // Numeric control
-    [[override(1300)]] let gain : f32;                     // Must be overridden
-    [[override]]       let width : f32 = 0.0;              // Specifed at the API level using
+    [[override(0)]]    let has_point_light: bool = true;  // Algorithmic control
```

Comment:
Nit: the comment starts don't line up anymore?
We can fix comment spacing after landing this.